samples/MvvmSampleUwp/MvvmSampleUwp/Assets/docs/ObservableObject.md

```diff
@@ -82,7 +94,10 @@ public class MyModel : ObservableObject
 }
 ```
 
-Here the `SetPropertyAndNotifyOnCompletion<TTask>(ref TTask, Expression<Func<TTask>>, TTask, string)` method will take care of updating the target field, monitoring the new task, if present, and raising the notification event when that task completes. This way, it's possible to just bind to a task property and to be notified when its status changes.
+Here the `SetPropertyAndNotifyOnCompletion<T>(ref TaskNotifier<T>, Task<T>, string)` method will take care of updating the target field, monitoring the new task, if present, and raising the notification event when that task completes. This way, it's possible to just bind to a task property and to be notified when its status changes. The `TaskNotifier<T>` is a special type exposed by `ObservableObject` that wraps a target `Task<T>` instance and enables the necessary notification logic for this method. The `TaskNotifier` type is also available to use directly if you have a general `Task` only.
```

Comment:
TaskNotifier isn't 'exposed by 'ObservableObject right? It's just meant to be 'that can be utilized by'?

Reply:
It is exposed, since users need to use them, it's just marked as `protected` so you only see that from inherited classes. I just said "exposed" here for simplicify but we can be more specific if you want 🙂